R/html-tweak.R

```diff
@@ -129,6 +129,103 @@ tweak_footnotes <- function(html) {
   xml2::xml_remove(container)
 }
 
+tablist_item <- function(tab, html, parent_id) {
```

Comment:
Could you give some high-level description of what's happening here? What powers the tabs? Is it bs and we're just providing some convenient markdown syntax for it?